Tutorials/Python_Developer_Tutorials/foundation/Commanding-Local/Readme.md

```diff
@@ -143,15 +162,7 @@ below is a screenshot of the partial output of the tutorial script when run in a
 
 The padding bytes are clearly visible appended to the end of the returned ECHO'd string. 
 
-#Bonus Excersize
+# Bonus Exercise
 
 * Modify the the example script to remove the padding bytes and display only the original ECHO'd string.
-* Can you do this variably given any string length using only the packets sent and received?
-  * *Hint: [Application layer](https://en.wikipedia.org/wiki/OSI_model) packet [encapsulation](https://en.wikipedia.org/wiki/Encapsulation_(networking))*
-
-#See Also
-
-* [Proxy Tool - FaradayIO](http://faraday-software.readthedocs.io/en/latest/faradayio.html)
-* [Proxy Tool - TelemetryParser](http://faraday-software.readthedocs.io/en/latest/telemetryparser.html)
-* Command Application
-
+  * Can you do this with variable length ECHO data?
```

Comment:
Your example doesn't show any extra text on the echo'd string. I don't understand.